src/data/real/nnreal.lean

```diff
@@ -85,7 +87,10 @@ begin
           add_comm_monoid.zero, add_comm, add_left_comm] }
 end
 
-instance : is_semiring_hom (coe : ℝ≥0 → ℝ) := by refine_struct {..}; intros; refl
+/-- Coercion `ℝ≥0 → ℝ` as a `ring_hom`. -/
+def coe_hom : ℝ≥0 →+* ℝ := ⟨coe, nnreal.coe_one, nnreal.coe_mul, nnreal.coe_zero, nnreal.coe_add⟩
```

Comment:
Can we call this `to_real` or something?

Reply:
```suggestion
def to_real_hom : ℝ≥0 →+* ℝ :=
⟨coe, nnreal.coe_one, nnreal.coe_mul, nnreal.coe_zero, nnreal.coe_add⟩
```